go.mod

```diff
@@ -6,21 +6,28 @@ require (
 	github.com/apache/thrift v0.13.0
 	github.com/bytedance/gopkg v0.0.0-20220817015305-b879a72dc90f
 	github.com/bytedance/mockey v1.2.0
+	github.com/bytedance/sonic v1.8.8
 	github.com/choleraehyq/pid v0.0.16
-	github.com/cloudwego/fastpb v0.0.4
+	github.com/cloudwego/dynamicgo v0.1.0-rc
+	github.com/cloudwego/fastpb v0.0.4-0.20230131074846-6fc453d58b96
```

Comment:
should not use commit for the lib has version tag

Reply:
fixed